src/Environment.cpp

```diff
@@ -168,9 +156,25 @@ bool startsWith(const std::string &str, const std::string &prefix)
 	return true; // All characters matched
 }
 
-// This function is used to set the CGI environment variables based on the request target.
-//  atm this is more of a debug function, especially as we implement only GET, POST and DELETE
-// or a draft for a future more advanced implementation
+// because at the moment we implement only GET, POST and DELETE methods
+// we don't need that, but we can add it later
+bool Environment::isAuthorityForm(const HTTPRequest &request)
+{
+	std::string method = request.getMethod();
+	std::string requestTarget = request.getRequestTarget();
+
+	if (method != "CONNECT")
+	{
+		return false;
+	}
+
+	if (requestTarget.find("://") != std::string::npos || requestTarget[0] == '/')
+	{
+		return false;
+	}
+	return true;
+}
+
 void Environment::RequestTargetToMetaVars(HTTPRequest request, Environment &env)
```

Comment:
In this method we have 6 if/else conditions but only 1 is doing something. Are the other ones going to be implemented in the future or for what do we need them?